db/migrate/20240416013510_add_attributes_to_reservable_items.rb

```diff
@@ -0,0 +1,15 @@
+class AddAttributesToReservableItems < ActiveRecord::Migration[7.1]
+  def change
+    add_column :reservable_items, :status, :enum, enum_type: "item_status"
+    add_column :reservable_items, :brand, :string
+    add_column :reservable_items, :model, :string
+    add_column :reservable_items, :serial, :string
+    add_column :reservable_items, :number, :integer
+    add_column :reservable_items, :purchase_price_cents, :integer
+    reversible do |direction|
+      direction.up { execute "UPDATE reservable_items SET number=id" }
+    end
+    change_column_null :reservable_items, :number, false
+    add_index :reservable_items, ["number", "library_id"], unique: true
```

Comment:
⚠️ Looks like power_source landed on item_pools but not reservable_items, but it moved to the Concern shared by all three models.

Reply:
I was thinking we wouldn't have items in an item pool with more than one power_source, although maybe that's incorrect? Currently `ItemAttributes` gets mixed into `ItemPool` and not `ReservableItem`, but due to the naming that's likely confusing. Definitely a WIP.

I wonder if it's worth doing a naming review at some point and seeing if we want to adjust what we're calling things before we write too much code using them?

Reply:
Ah ok I missed that ItemAttributes wasn't mixed into ReservableItem, that clears up my concern! I'm not super concerned with doing a naming review earlier vs later, but I do think it would be good to do one before we go live.